azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/error/ErrorHandler.kt

```diff
@@ -86,7 +87,9 @@ internal class ErrorHandler(
                     getCallCompositeErrorCode(callStateError.errorCode),
                     null,
                 )
-            configuration.callCompositeEventsHandler.getOnErrorHandlers().forEach { it.handle(eventArgs) }
+            if (eventArgs.errorCode != null) {
```

Comment:
Why the new check? Can errorCode == null?

Is there callStateErrors that return a null error code? Might be a good candidate for an "unknown" error code to make non-nullable.

Reply:
yes. ErrorCode can be null in cases where we propagate internal errors through error handler.